### PR TITLE
Change "Run Script" to "Run radare2 script"

### PR DESCRIPTION
**Pull request by ITAYC0HEN**
**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://cutter.re/docs/contributing/code/getting-started.html) to this repository
- [x] I made sure to follow the project's [coding style](https://cutter.re/docs/contributing/code/development-guidelines.html)
- [x] I've updated the [documentation](https://cutter.re/docs/user-docs.html) with the relevant information (if needed)

**Detailed description**

This pull request updates a menu item text to make it clearer for the reader

**Test plan (required)**

Go to File menu and make sure the action says "Run radare2 script" and not "Run Script".
Make sure that it works and indeed running a script.
Make sure that the docs are updated correctyl

**Related issues**

Related to #2271

### CODE REVIEW

**Comment by karliss** [comment]
Maybe the documentation could have one more sentence explaining that r2 script is file containing r2 commands or a link to r2 doc showing commands.

File: docs/source/user-docs/menus/menu-bar/file-menu.rst [hunk 1]
```diff
@@ -78,9 +78,17 @@ Export to code
 
 Run a radare2 script
 ----------------------------------------
-**Description:** Cutter allows you to execute radare2 scripts to automate task or transfer information.   
+**Description:** Cutter allows you to execute radare2 scripts to automate task or transfer information. Radare2 scripts are files that contain list of radare2 commands. The scripts can be created manually by you, or automatically generated by radare2 commands (which are typically end with a star character, i.e ``afl*``).
```
Comment (Surendrajat) [inline docs/source/user-docs/menus/menu-bar/file-menu.rst:81]:
```suggestion
**Description:** Cutter allows you to execute radare2 scripts to automate task or transfer information. Radare2 scripts are files that contain list of radare2 commands. The scripts can be created manually by you, or automatically generated by radare2 commands (which typically end with a star character, for eg. ``afl*``).   
```

**Review by surendrajat — CHANGES_REQUESTED** [review]

File: docs/source/user-docs/menus/menu-bar/file-menu.rst (hunk 1 above)
Comment (ITAYC0HEN) [reply docs/source/user-docs/menus/menu-bar/file-menu.rst:81]: "character. e.g ``afl*``"

**Review by karliss — APPROVED** [review]

File: docs/source/user-docs/menus/menu-bar/file-menu.rst (hunk 1 above)
Comment (ITAYC0HEN) [inline docs/source/user-docs/menus/menu-bar/file-menu.rst:81]:
```suggestion
**Description:** Cutter allows you to execute radare2 scripts to automate task or transfer information. Radare2 scripts are files that contain list of radare2 commands. The scripts can be created manually by you, or automatically generated by radare2 commands (which are typically end with a star character. e.g. ``afl*``).   
```

File: docs/source/user-docs/menus/menu-bar/file-menu.rst [hunk 2]
```diff
@@ -78,14 +78,22 @@ Export to code
 
 Run a radare2 script
 ----------------------------------------
-**Description:** Cutter allows you to execute radare2 scripts to automate task or transfer information.   
+**Description:** Cutter allows you to execute radare2 scripts to automate task or transfer information. Radare2 scripts are files that contain list of radare2 commands. The scripts can be created manually by you, or automatically generated by radare2 commands (which are typically end with a star character. e.g. ``afl*``).
```
Comment (ITAYC0HEN) [inline docs/source/user-docs/menus/menu-bar/file-menu.rst:81]:
```suggestion
**Description:** Cutter allows you to execute radare2 scripts to automate task or transfer information. Radare2 scripts are files that contain list of radare2 commands. The scripts can be created manually by you, or automatically generated by radare2 commands (which typically end with a star character. e.g. ``afl*``).   
```

**Review by surendrajat — APPROVED** [review]

**Pull request merged**